src/main/scala/edu/twitter/model/SentimentModelDataCreator.scala

```diff
@@ -18,20 +18,12 @@ class SentimentModelDataCreator(tweetsRDD: RDD[Row]) {
     * @return training data set, testing data set
     */
   def getTrainingAndTestingData(): (RDD[LabeledPoint], RDD[LabeledPoint]) = {
```

Comment:
can you change the documentation of this method

Reply:
fixed :+1: